### ISSUE TITLE
Facilities -  Sprint 57

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#8835]
<details><summary>Background</summary>

🥅   — Sprint goals (pending Sprint planning)
💡 — Design
💧  — Drupal engineering
🎉 — FE
🎓  — Training / Change management / KB
📟 — Helpdesk
🔮  —  Planning / roadmap / epic refinement
📊  —  Analytics
🚰  — Service design
🌱 — Good first task

</details>

## Facility Locator
- [ ]  department-of-veterans-affairs/va.gov-team-sensitive#454 (Needs new issues created to reflect the plan in [this slack thread](https://dsva.slack.com/archives/C0FQSS30V/p1650660997660989)) 🥅
- [ ] Get to know FL via #8869 (Max/Ethan?)

<details><summary>Lower priority candidates for Q2</summary>

- [ ] #8870
- [ ] Bounding box to Radius got rolled back to bounding box, possibly low priority to go back to radius.
- [ ] Get to know Provider Profile Management System - Data source for Community care providers, Urgent Care, Community pharmacies. VAOS (VA Online Scheduling) also consumues this endpoint in Vets API

</details>

## #7622 Lovell

Background: See epic https://github.com/department-of-veterans-affairs/va.gov-team/issues/34022

### Front-end

Some  design background is in https://github.com/department-of-veterans-affairs/va.gov-team/issues/38196
- [x] #8715 
- [ ] #8716 (Val) 🥅
- [ ] #8862 (Suzanne)

<details><summary>CMS (on pause until design is validated)</summary>

- [x] #7626 (sprint 55)
- [ ] Create proof of concept
- [ ] CMS design discovery epic
  - [ ] #8863
- [ ] Refine epic #1939 (for DoD facilities)
  - [ ] #1940 
  - [ ] #1941 
  - [ ] #1942 
  - [ ] #1943  

</details> 

### #6931 Deprecate table-based hours in favor of office hours field
(_These appear farther down in lighthouse push section as they are entangled there._)
Which issues are ready to go here? Do we need some updates to training and/or change management?
- [x] https://github.com/department-of-veterans-affairs/va.gov-team/issues/39436
- [ ] #8708
- [x] #8550
- [x] #8709  (PR ready)
- [x] #8426
- [ ] #8861 

Discuss if action is needed:
- [ ] Update KB article
- [x] Change management - a dozen or so instances where hours data was not migrateable. (?) 🥅

###  #3151 3 VAMC Top task pages

Front-end (epics) ... Need updating
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/37367
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/37365
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/37366

CMS 
- [x] #8337 
- [x] #8648 

## Events V2 and Recurring events

Issues in epic https://github.com/department-of-veterans-affairs/va.gov-team/issues/28918

This epic is owned by Public Websites but could be contributed to by Ethan who has context, pending capacity.

### #7624 Support lighthouse with unique ID for health service terms / #5017  Push facility services to lighthouse

<details><summary>Blocked by Lighthouse v1 API</summary>

- [ ] #8019 done but not merged, blocked by Lighthouse 
- [ ] #8017  blocked by Lighthouse AND
    - [ ] Field Office Hours hardening https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6931
      - [x] script the migration https://github.com/department-of-veterans-affairs/va.gov-cms/pull/8533
      - [x] run the migration on prod to populate data for FE 
      - [ ] FE builds new display  https://github.com/department-of-veterans-affairs/va.gov-team/issues/39436 
      - [x] After FE goes live(same day) Hide field `field_facility_service_hours` and show 'field_office_hours' in cms  #8550 
      - [x] Re-run script on prod `drush scr scripts/content/VACMS-6931-switch-service-locations-office-hours.php`
      - [ ] Notify editors / process bad entries from the outliers.csv #8708
      - [x] #8709

Possible to add to Lighthouse V0 API
- [x] Stretch: #8565 
- [x] Stretch: #7726
- [x] Stretch: #7279
- [x] Stretch: #6318
- [ ] Stretch: #4769 

</details>

## Operations & Maintenance (Steve, Jay, User support team)

- [ ] #7728 and its many children

**Comment by RWEagans** [comment]
Covid banner issue will push some of this work.
